opencadd/structure/superposition/cli.py

```diff
@@ -35,26 +35,20 @@ def parse_cli(argv=None, greet=False):
         help="Which alignment method to use",
         choices=[m.lower() for m in METHODS],
     )
-    # p.add_argument(
-    #     "--method-options",
-    #     default={},
-    #     type=parse_method_options,
-    #     help="Options to be passed to the chosen method. Syntax is `key: value; key: value;...`",
-    # )
+    p.add_argument(
+        "--method-options",
+        default={},
+        type=parse_method_options,
+        help="Options to be passed to the chosen method. Syntax is `key: value; key: value;...`",
```

Comment:
How would the user find out which key-value pairs are possible for a given method?

Reply:
The method options are described for each method in the docs.
Example for mda: [mda api docs](https://opencadd.readthedocs.io/en/latest/generated/opencadd.structure.superposition.engines.mda.MDAnalysisAligner.html)

I think it would be too much to print it in the cli.